### PR TITLE
[MM-57958] Calls: Host controls: stop screenshare, mute, lower hand

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Next three host controls for mobile: stop screenshare, mute, lower hand
- Mobile only has to obey the mute and lower hand command (it can't screenshare)

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-57958

#### Checklist
- [ ] Added or updated unit tests (required for all new features)
- [x] Has UI changes
- [x] Includes text changes and localization file updates
- [x] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
- Android: 13, Pixel 6
- Android: 13, Galaxy Tab s7+
- iOS: 16.5.1, iPhone 14

#### Release Note
```release-note
Calls: Host controls: stop screenshare, mute, lower hand
```

### CODE REVIEW

File: app/products/calls/screens/host_controls/host_controls.tsx [hunk 1]
```diff
@@ -43,19 +43,40 @@ export const HostControls = ({
     const intl = useIntl();
     const {bottom} = useSafeAreaInsets();
 
+    const sharingScreen = currentCall.screenOn === session.sessionId;
+
     const makeHostPress = useCallback(async () => {
-        await makeHost(currentCall.serverUrl, currentCall.channelId, session.userId);
+        await hostMake(currentCall.serverUrl, currentCall.channelId, session.userId);
```
Comment (enahum) [inline app/products/calls/screens/host_controls/host_controls.tsx:49]: Do we want to wait for the action and then dismiss the bottom sheet? Normally is the other way around, we dismiss the sheet and then perform the action cause if it fails the sheet will remain opened and there is no feedback anyway, causing a bit of confusion imho

**Review by enahum — APPROVED** [review]

File: app/products/calls/screens/host_controls/host_controls.tsx (hunk 1 above)
Comment (cpoile) [reply app/products/calls/screens/host_controls/host_controls.tsx]: Yep, makes sense, done. Thanks

File: app/products/calls/screens/host_controls/host_controls.tsx [hunk 2]
```diff
@@ -43,19 +43,40 @@ export const HostControls = ({
     const intl = useIntl();
     const {bottom} = useSafeAreaInsets();
 
+    const sharingScreen = currentCall.screenOn === session.sessionId;
```
Comment (streamer45) [inline app/products/calls/screens/host_controls/host_controls.tsx:46]: Unrelated just find it a bit confusing as the variable name suggests a boolean when in fact it's the session id of the presenter.

**Review by streamer45 — APPROVED** [review]
Looks good!

File: app/products/calls/screens/host_controls/host_controls.tsx (hunk 2 above)
Comment (cpoile) [reply app/products/calls/screens/host_controls/host_controls.tsx:46]: Totally agree, I'm confused too. I think it was a leftover from an old naming scheme "who is the sceenOn" maybe?

**Pull request merged**